prediction/src/algorithms/evaluation/evaluate_detection.py

```diff
@@ -311,13 +311,13 @@ def compare_results(detections, annotations):
     df.to_csv(detection_detail_file, index=False)
 
 detections = filter_detections(prediction, detection)
-print 'number of detections:', len(detections)
+print('number of detections:', len(detections))
 
 annotations = load_annotations_OsiriX(annotation_path)
-print 'number of annotations:', len(annotations)
+print('number of annotations:', len(annotations))
 
 correct_detection = compare_results(detections, annotations)
-print 'number of correct_detection:', correct_detection
```

Comment:
Odd, how did this work before? I mean, Py2 vs Py3 etc.

Reply:
It didn't :) Even more, I don't understand the purpose of `evaluate_detection.py`, 'cause in PR #301:
> We extend the IoU calculation from 2D to 3D but in order to be simple, we only deal with 3D boxes / cubes. Although the detections we have are 3D spheres output from the grt123 system. The threshold we currently use is 0.5 but can easily be changed (‘correct_detection_threshold’ in evaluate_detection.py). We also did some experiments using different thresholds, please check the evaluation results below.  

The thing is that do the same for 3D spheres even easier 'cause you have to deal with centroids and diameters only (which are given) no need to compute bboxes. That [evaluation was proposed by the LUNA16 authors](https://luna16.grand-challenge.org/evaluation/) and was adjusted in PR #292

Reply:
@lamby, [Codestyle of evaluate_detection.py](https://github.com/concept-to-clinic/concept-to-clinic/pull/307/commits/19ed5779483a3dad25f7c8fd38239d77a9dc06d0) commit moved to PR #307 as was [requested](https://github.com/concept-to-clinic/concept-to-clinic/pull/307#issuecomment-362267602) by @reubano